test-it/testList.json

```diff
@@ -8,6 +8,7 @@
                 "ListDirectory.via",
                 "Round.via",
                 "Scale2X.via",
+                "Scale2XWithIntegers.via",
```

Comment:
If I remember correctly we put it here since it was failing on appveyor, you might ask Milan about it, meanwhile I'm just going to put it here, like the Scale2.via. We can move it later.